tox.ini

```diff
@@ -3,8 +3,8 @@ isolated_build = true
 envlist =
     docs
     packaging
-    py{38,39,310,311,312}-dj{42}-{sqlite,postgresql,postgis,mysql}
-    py{310,311,312}-dj{42,50,51,main}-{sqlite,postgresql,psycopg3,postgis,mysql}
+    py{38,39,310,311,312,313}-dj{42}-{sqlite,postgresql,postgis,mysql}
+    py{310,311,312,313}-dj{42,50,51,main}-{sqlite,postgresql,psycopg3,postgis,mysql}
```

Comment:
No version of Django before 5.1.3 support Python 3.13 if I interpret this recent change correctly:

https://github.com/django/django/commit/2e3bc59fd3760de87952ec8fd6cd3694e8d9dc1c

So, 313 should really only be added to dj51 (once 5.1.3 is actually released) and djmain for now.

Reply:
https://forum.djangoproject.com/t/backport-python-3-13-support-in-django-5-0/34671 better source.

Reply:
I've separated out 3.13 to match up with expected Django compatibility.